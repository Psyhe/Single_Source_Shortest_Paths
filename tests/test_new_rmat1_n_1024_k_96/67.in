1024 734 743
736 145 89
736 129 180
736 65 254
736 4 41
736 390 145
736 19 240
736 550 111
736 16 208
737 0 237
737 72 142
738 936 145
740 64 145
740 528 94
740 1 148
740 131 58
740 388 21
740 276 99
740 82 233
740 2 87
740 128 158
741 1 154
742 385 247
743 133 160
