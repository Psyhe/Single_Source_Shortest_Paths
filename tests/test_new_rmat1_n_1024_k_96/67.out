1000000000000000000
1000000000000000000
16
43
101
249
69
185
245
1000000000000000000
